Source/GoBot/ViewDatabase.swift

```diff
@@ -1578,11 +1578,18 @@ class ViewDatabase {
                     }
                 }
                 
+                /* This is the don't put older than 6 months in the db. 
```

Comment:
Does this mean that we will put older that 6 months content in the db? In that case, why do we want this?

Reply:
well i was. noticing that it wasn't updating / setting about messages because of this. I agree we might not want this.